contracts/TablelandTables.sol

```diff
@@ -102,20 +104,38 @@ contract TablelandTables is
     function _getPolicy(
         address caller,
         uint256 tableId
-    ) private returns (ITablelandController.Policy memory) {
+    ) private returns (Policy memory) {
         address controller = _controllers[tableId];
         if (_isContract(controller)) {
-            return
-                ITablelandController(controller).getPolicy{value: msg.value}(
-                    caller
-                );
+            ITablelandControllerV2 controllerV2 = ITablelandControllerV2(
+                controller
+            );
+
+            // Check if using new Controller interface
+            if (address(controllerV2) != address(0)) {
```

Comment:
so, `address(controllerV2) != address(0)` will be true if the interfaces don't match?

Reply:
Been working on the finishing the tests, and this seems _not_ to be the case. I've converted this PR to draft, since I'll have to figure another way to maintain backward compatibility, maybe with a try / catch block. I feel like there has to be a more elegant solution, so I'm searching.